composer.json

```diff
@@ -40,6 +40,7 @@
         "symfony/dotenv": "^6.4|^7.0",
         "symfony/maker-bundle": "^1.55",
         "symfony/phpunit-bridge": "^6.4|^7.0",
+        "symfony/runtime": "^6.4|^7.0",
```

Comment:
I've added runtime as a dev requirement because the fix [only works with runtime present](https://github.com/symfony/symfony/pull/58372/files#r1774895155)